### ISSUE TITLE
Make sure authentication continues to work on spineynorman after February 13th

### ISSUE THREAD

**Issue by ljpearlman** [informatics-isi-edu/cirm-project#3]
This host uses the older globusonline provider. Kyle said that he was told that it should continue to work and that another group has tested it.

I've been able to test this once (this is a production server, so testing has to be scheduled). I got a globus token, but the login process failed trying to get my detailed user/group information. It's possible this is something specific to my account, as I was just added to the cirm-usc group today, and it's possible that hasn't propagated to the beta server.

I've asked Seth for another test window tomorrow. I may need help from someone who's been in the cirm-usc group longer than I have.

**Comment by ljpearlman** [comment]
Between Seth and Kyle, I got added to the group. I haven't been able to test it yet, because I haven't heard back from Seth about scheduling (I sent him another email).

This system doesn't use webauthn for initial authentication; instead, the http server is set up with a proxy to the Globus server, and the web application code sends the user to that proxy and stores the user's access token in a cookie with a one-week lifetime. I think this could be a problem after February 13th for users who have unexpired cookies from the current server. These cookies persist across browser shutdowns, so if this does turn out to be a problem, users will have to delete them manually (or wait for them to expire).This isn't something we can test in advance -- cookies issued by the existing server shouldn't work for the beta server in any case.
